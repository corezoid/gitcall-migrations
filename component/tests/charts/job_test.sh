#!/usr/bin/env bash
$CZ helm template | sed -n '/job\.yaml/,/Source\:/p'
