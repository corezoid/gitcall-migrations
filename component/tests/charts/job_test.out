# Source: gitcall_migrations_release/charts/gitcall_migrations/templates/job.yaml
apiVersion: batch/v1
kind: Job
metadata:
  name: RELEASE-NAME-gitcall-migrations-job
  annotations:
    "helm.sh/hook": pre-install,pre-upgrade
    "helm.sh/hook-delete-policy": hook-succeeded
  labels:
    release: "RELEASE-NAME"
    app: "gitcall"
    tier: "migrations"
spec:
  activeDeadlineSeconds: 180 # 3min
  ttlSecondsAfterFinished: 360 # 6min
  template:
    metadata:
      labels:
        release: "RELEASE-NAME"
        app: "gitcall"
        tier: "migrations"
    spec:
      restartPolicy: Never
      containers:
      - name: main
        image: docker-hub.middleware.biz/gitcall/gitcall-migrations:aVersion
        env:
          - name: MIGRATOR_USERNAME
            value: "aMigratorUsername"
          - name: MIGRATOR_PASSWORD
            value: "aMigratorPassword"
          - name: DB
            value: "aGitcallDb"
          - name: HOST
            value: "aDbHost"
          - name: PORT
            value: "5555"
          - name: DIR
            value: "migrations/gitcall"
